services/common/src/constants/API.ts

```diff
@@ -64,6 +64,11 @@ export const PERMIT_AMENDMENT = (mineGuid, permitGuid, permitAmendmentGuid) =>
 
 export const PERMIT_AMENDMENT_VC = (mineGuid, permitGuid, permitAmendmentGuid) =>
   `/mines/${mineGuid}/permits/${permitGuid}/amendments/${permitAmendmentGuid}/verifiable-credential`;
+export const VERIFIABLE_CREDENTIALS_CONNECTIONS = (partyGuid: string) =>
```

Comment:
This endpoint returns all issuance records for mines-act-permits. so this name is a bit misleading.

Reply:
Fair enough.  Some of the naming conventions are still a bit foreign to me.  Something like VC_ISSUANCE_RECORDS?

Reply:
VC_ISSUANCE_RECORDS sounds clear to me, but I will wait for @Jsyro take on it.

Reply:
how about `MINES_ACT_PERMITS_VC_LIST` not exactly sure how that jives with the conventions.